qfieldsync/gui/package_dialog.py

```diff
@@ -61,6 +61,8 @@
 
 
 class PackageDialog(QDialog, DialogUi):
+    MAX_LENGTH_CHARS_FILEPATH = 200
```

Comment:
Better make it a constant outside the class. In theory can be used in other places besides the class. Just move it righth after imports in the top of the file.